cpp/oneapi/dal/algo/linear_regression/backend/model_impl.hpp

```diff
@@ -22,43 +22,33 @@
 
 namespace oneapi::dal::linear_regression {
 
+using norm_eq_proto = ONEDAL_SERIALIZABLE(linear_regression_model_impl_id);
```

Comment:
The same question: why the ID linear_regression_norm_eq_model_impl_id was changed to linear_regression_model_impl_id?
If we would add a model for QR compute method, for example, that model might have different internal structure than norm_eq model, so it will need a different serialization ID.

Reply:
Just because all models can be converted to the form: intercept + coefficients

Reply:
Discussed